### PR TITLE
Tele bot

### PR DESCRIPTION
**Pull request by UmangDiyora**

### CODE REVIEW

**Comment by UmangDiyora** [comment]
Telegram Skeleton

**Comment by UmangDiyora** [comment]
telegram signup and login with telegram collection in Database

File: Bot/Telegram/tmp.ipynb [hunk 1]
```diff
@@ -0,0 +1,95 @@
+{
```
Comment (abhira0) [inline Bot/Telegram/tmp.ipynb:1]: remove this file

File: Bot/Telegram/bot.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
+import os
```
Comment (abhira0) [inline Bot/Telegram/bot.py:1]: run pre-commit on this file, and fix errors.

Comment (abhira0) [inline Bot/Telegram/bot.py:1]: also, add testcases

**Review by abhira0 — CHANGES_REQUESTED** [review]

**Comment by abhira0** [comment]
rename the folder names to lower case!

**Pull request merged**